.github/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 3. Bug - Fixed OCAS region coordinates abeam Boscombe Down (Cotswold CTA) - thanks to @hazzas-99
 4. Bug - Corrected Biggin Hill (EGKB) STARs following WAD (removed BEDEK and added SIRIC) - thanks to @hazzas-99
 5. Bug - Fixed Lakes/North Sea Alternative sector ownership for observing/mentoring - thanks to @hazzas-99
+6. Enhancement - Added Bristol (EGGD) Stand 18 and renamed stands 13-15 to 14-16 - thanks to @rishab-alt
```

Comment:
```suggestion
6. AIRAC (2304) - Added Bristol (EGGD) Stand 18 and renamed stands 13-15 to 14-16 - thanks to @rishab-alt
```
This is an AIRAC change not an enhancement :) (Usually the changelog type is the same as the issue tag)